### DISCUSSION TITLE
How do I open the software?!?!?

### DISCUSSION THREAD

**Discussion by Gorilli09** [vgmtrans/vgmtrans#281] [General]
I can't find the .exe file for this!

**Comment by sykhro** [comment]
Actions -> click on the latest workflow -> scroll down to artifacts. Or download it from from [here](https://vgmtrans.github.io/vgmtrans/)

**Reply by Gorilli09** [reply]
That didn't help! Why didn't anyone upload a tutorial to this new software?!

**Reply by sykhro** [reply]
Sorry, try again, I had a typo in the link
